cob-stud-clients/sync_packages.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
please consider updated scripts for robot as proposed in https://github.com/ipa320/setup_cob4/pull/84

is it possible to make the `pcs` list configurable from outside of the script and use the functional part for both robots and cob-stud-pcs.... :arrow_right: harmonization and less maintenance...